index.html

```diff
@@ -202,47 +177,41 @@ <h2>Photo Gallery</h2>
-         <br><br>
-         <label>What is your email ?:<br>
-          <input type="email" name="email_field"></label>
-          <br><br>
-          <label>Which integer is best ?:<br>
-        <input type="number" name="best_integer"></label>
-        <br><br>
-        <label>Which day is best ?:<br>
-        <input type="date" name="best_day"></label>
-        <br><br>
-        </form>
-        <label>Which bear is best ?:<br>
-        <input list="bears" name="best_bear" ></label>
-        <datalist id="bears">
-        <option value="none">
-        <option value="Black">
-        <option value=" Brown">
-        <option value="Care">
-        <option value="Panda">
-        <option value="Polar">
-        <option value="Teddy">
-        </datalist> 
-        <br><br>
-        <label>Do you like radio buttons ?:<br >
-        <label for="Yes">Yes</label>
-        <input type="radio" name="yes" value="yes"><br>
-        <label for="No">No</label>
-        <input type="radio" name="no"value="no"><br>
-        <label for="None of your business"> None of your business</label>
-        <input type="radio" name="?"value="?" ><br></label>
-        <br><br>
-        What is the meaning of life ?:
-        <textarea rows="10" cols="85">
-        Enter the correct answer here
+         <form action="?" method="POST">
+         <label for="name_field">What is your name ?:</label>
+         <input type="text" name="name_field" id="name_field" required>
+         <label for="email_field">What is your email ?:</label>
+          <input type="email" name="email_field" id="email_field" required>
+          <label for="best_integer">Which integer is best ?:</label>
+        <input type="number" name="best_integer" id="best_integer">
+        <label for="best_day">Which day is best ?:</label>
+        <input type="date" name="best_day" id="best_day">
+        <label for="best_bear">Which bear is best ?:</label>
+        <select name="best_bear" id="best_bear">
+            <option value="none">
+            <option value="Black">
+            <option value=" Brown">
+            <option value="Care">
+            <option value="Panda">
+            <option value="Polar">
+            <option value="Teddy">
+        </select> 
+        <label>Do you like radio buttons ?:</label>
+        <label>Yes
+            <input type="radio" name="group1" value="yes">
+        </label>
+        <label>
+            No
+            <input type="radio" name="group1" value="no">
+        </label>
+        <label> 
+            None of your business
+            <input type="radio" name="group1" value="?" >
+        </label>
+        <label for="life_meaning">What is the meaning of life ?:</label>
+        <textarea rows="10" cols="85" name="life_meaning" id="life_meaning" placeholder="Enter the correct answer here">
         </textarea>
-        <div class="Send">
-        <input type="button" name="Send" value="Send">
-        </div>
```

Comment:
No need for the wrapping div.  Just use an appropriate selector such as [type="submit"] in your CSS
The type needs to be 'submit' so that the form submits when the button is clicked